### DISCUSSION TITLE
"My discussions" repository link

### DISCUSSION THREAD

**Discussion by shaedrich** [community/community#72166] [Discussions]
### Select Topic Area

Product Feedback

### Body

It would be nice if, when I click on the repository link at https://github.com/discussions, I'd be navigated to that repository's discussions rather than it's general overview

**Comment by Unmesh100** [comment]
Your suggestion addresses a small but impactful enhancement to GitHub's user experience. Redirecting users directly to the Discussions section of a repository when they click on the repository link at https://github.com/discussions would streamline navigation, especially for those specifically interested in participating in or reviewing discussions.

Your proposal:

Direct Link to Discussions – Instead of navigating users to the repository's general overview, the link would take them straight to the Discussions tab, saving a click and reducing the need for users to manually navigate to that section.
This would improve the user flow for those who frequently engage with Discussions, making the platform more intuitive and efficient for community interaction. Have you considered sharing this with GitHub's support or feedback channels? This could be a well-received improvement among users who actively participate in Discussions!

**Reply by shaedrich** [reply]
This answer is AI-generated, isn't it?

**Reply by LiteBrite82** [reply]
Hi @Unmesh100 , 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.
